### DISCUSSION TITLE
wsl set default version

### DISCUSSION THREAD

**Discussion by goofyclown** [microsoft/WSL#6495] [General]
okay so i've tried to get wsl on windows insider at start but didnt seemed to work so i did manual thing and I got thing when i execute the "wsl --set-default-version 2"

here's the error log (french):

PS C:\WINDOWS\system32> wsl --set-default-version 2
Pour plus d’informations sur les différences de clés avec WSL 2, visitez https://aka.ms/wsl2

i dont know if that mean version is set to 2, i've also tryed enable VT on my bios, i've check marked it and saved things but when i execute "(GWMI Win32_Processor).VirtualizationFirmwareEnabled" in PS it says "False"
i use a HP pro book

**Comment by adanribeiro** [comment]
Have you already read [this post](https://github.com/microsoft/WSL/issues/6486)?
